README.md

```diff
@@ -5,6 +5,19 @@ The template used by `npx @react-native-community/cli init` to bootstrap a React
 [![Build Status][build-badge]][build] [![Version][version-badge]][package] [![MIT License][license-badge]][license] [![PRs Welcome][prs-welcome-badge]][prs-welcome]
 
 
+# Create Latest CLI Project
+Using this command create latest react-native project with templete
+```
+npx @react-native-community/cli init TestApp
+```
+
+# Create Specific Version React-Native CLI Project
+add version flag and specify that existing version of react-native to create project
+
+```
+npx @react-native-community/cli init TestApp --version 0.72.8
+```
```

Comment:
This is confusing IMHO. We should either clarify that this uses a specific React Native version OR add an instruction for the `--template` property. @szymonrybczak what do you think?

Reply:
Yea, I think we should add callout that it's using a specific version of React Native and user should probably init with latest, and also I'd skip adding hardcoded number to avoid situation when beginners will start copying and sharing this command. I'd go with something like this:
```
npx @react-native-community/cli init TestApp --version VERSION
```